bigtop-tests/spec-tests/runtime/src/test/resources/testRuntimeSpecConf.groovy

```diff
@@ -218,7 +218,7 @@ specs {
     }
     'HADOOP_BINCONTENT_COMMON' {
       name = 'HADOOP_BINCONTENT_COMMON'
-      type = 'dircontent'
+      type = 'dirstruct'
```

Comment:
I don't understand what changing the type from dircontent to dirstruct does, nor why it's important.